README.md

```diff
@@ -1,38 +1,121 @@
 [![Depfu](https://badges.depfu.com/badges/3510c0fc8a9b17a045ef2389a436fe9d/count.svg)](https://depfu.com/github/theianjones/shoulditraintoday?project_id=27149)
 
-This is an opinonated Next.js starter project that makes it relatively simple to spin up a new project.
+# ![](https://res.cloudinary.com/dpspogkzf/image/upload/v1618869726/shoulditrain/logo_wl1ezx.svg) shoulditrain.today
 
-## Getting Started
+This project is based on a [quiz from a local (Northern VA, USA) nutrition company](https://www.instagram.com/p/CL7kiaqnP80/). Essentially, it is an 8 question quiz where the user must self-assess their answers. We then apply points to the answers that the user gives. At the end of the quiz, we perform a calculation and give a "% of readiness" for the user to expect from themselves.
 
-First, run the development server:
+This number is by no means definitive after taking the quiz consistently, it would give you a good baseline of how ready you feel to train on a given day.
```

Comment:
The phrasing here is a little confusing.

-> This number is by no means definitive but after taking the quiz consistently you should see what a good baseline recovery is for yourself on any given day. This number will represent how ready you feel to train on that day